setup.py

```diff
@@ -60,6 +60,9 @@
     "tox",
 ]
 
+# *1: Ops 2.0 requires Python 3.8, so we can only upgrade once we can
+# guarantee we'll have it. Currently, spread tests on bionic use Python 3.6.
```

Comment:
if the issue is only spread tests on bionic, could we conceivably use the deadsnakes ppa to install 3.8 there?